Specs/Core/PolylineGeometrySpec.js

```diff
@@ -31,13 +31,13 @@ defineSuite([
         }).toThrowDeveloperError();
     });
 
-    it('constructor throws with invalid width', function() {
+    it('constructor does not throw with invalid width', function() {
```

Comment:
Rather than changing this text not to throw, we should instead delete this test and add a test where `PolylineGeometry.createGeometry` is called and returns undefined.  Basically, copy `constructor computes all vertex attributes` around line 52 and change width to -1, then remove all the expects except for `line` to be undefined.